snode/src/main/java/org/apache/rocketmq/snode/SnodeController.java

```diff
@@ -121,52 +114,53 @@
     private PushService pushService;
     private ClientService clientService;
     private SlowConsumerService slowConsumerService;
+    private WillMessageService willMessageService;
 
     private final ScheduledExecutorService scheduledExecutorService = Executors
-        .newSingleThreadScheduledExecutor(new ThreadFactoryImpl(
-            "SnodeControllerScheduledThread"));
+            .newSingleThreadScheduledExecutor(new ThreadFactoryImpl(
+                    "SnodeControllerScheduledThread"));
 
     public SnodeController(ServerConfig nettyServerConfig,
-        ClientConfig nettyClientConfig,
-        SnodeConfig snodeConfig) {
+            ClientConfig nettyClientConfig,
+            SnodeConfig snodeConfig) {
         this.nettyClientConfig = nettyClientConfig;
         this.nettyServerConfig = nettyServerConfig;
         this.snodeConfig = snodeConfig;
         this.enodeService = new EnodeServiceImpl(this);
         this.nnodeService = new NnodeServiceImpl(this);
         this.scheduledService = new ScheduledServiceImpl(this);
         this.remotingClient = RemotingClientFactory.getInstance().createRemotingClient()
-            .init(this.getNettyClientConfig(), null);
+                .init(this.getNettyClientConfig(), null);
         this.mqttRemotingClient = RemotingClientFactory.getInstance()
-            .createRemotingClient(RemotingUtil.MQTT_PROTOCOL)
-            .init(this.getNettyClientConfig(), null);
+                .createRemotingClient(RemotingUtil.MQTT_PROTOCOL)
+                .init(this.getNettyClientConfig(), null);
 
         this.sendMessageExecutor = ThreadUtils.newThreadPoolExecutor(
-            snodeConfig.getSnodeSendMessageMinPoolSize(),
```

Comment:
@xiangwangcheng it seems that you modified some codes that has nothing to do with this function, and just modified the code format, which will make the code review a little difficult. Could you download a rocketmq code style from our website: https://rocketmq.apache.org/docs/code-guidelines/?

Reply:
@duhenglucky OK. I will apply the rmq code style.